docker/robot/Makefile

```diff
@@ -1,16 +1,32 @@
 
 # Building docker image
-VERSION = "v1.8.1" 
+VERSION = "v1.8.1"
```

Comment:
This variable should stay as is, because the Version will be in step with the ROBOT, not ODK, versions.

Reply:
Agreed. Here I just removed the trailing whitespace – just to satisfy my OCD about this kind of things...

Reply:
Yeah I didnt mean to say anything about the whitespace, just to mention that the versioning for the ROBOT image works somehow different from the versioning for the others! Thank you!